src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeRepositoryTest.java

```diff
@@ -45,7 +45,8 @@ void should_save_and_read_envelope_by_id() {
             "hello.zip",
             now(),
             now().plusSeconds(100),
-            DISPATCHED
+            DISPATCHED,
+            1024L
```

Comment:
may be we can test null filesize as well?

Reply:
When null is inserted as fileSize - 0L is returned in select result in integration test.
I assume this is because of using

spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver

in tests.

I am thinking about using 0L instead of null in existing calls of 

new NewEnvelope(...)

to avoid this confusion.